src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt

```diff
@@ -100,6 +101,7 @@ object Messages {
         "Hello, %name%, I don't suppose you're a tester? Mods like to see a tester around, it gives them something to aim at.",
         "%name% has a mind that ticks like a clock and, like a clock, it regularly goes cuckoo!",
         "%name% gives their compliments to the mods, and begs them to keep their abnormally large noses out of other people's business."
+        "Have you ever heard the tragedy of %name% the Wise? I hought not. It's not a story the mods would tell you."
```

Comment:
You have a typo here! "thought" not "hought"

Reply:
Fixed!